packages/pg-v5/commands/repoint.js

```diff
@@ -0,0 +1,50 @@
+'use strict'
+
+const cli = require('heroku-cli-util')
+const co = require('co')
+
+function * run (context, heroku) {
+  const host = require('../lib/host')
+  const util = require('../lib/util')
+  const fetcher = require('../lib/fetcher')(heroku)
+  let { app, args, flags } = context
+  let db = yield fetcher.addon(app, args.database)
```

Comment:
Does this allow repointing followers to leaders on other apps? If not, is that something we need/want to support?